tests/src/Unit/Traits/SettingsDataTrait.php

```diff
@@ -78,4 +78,14 @@ private function getValidVisibilitySettings() {
       'path_patterns' => "/admin\n/admin/*\n/batch\n/node/add*\n/node/*/*\n/user/*/*\n/block/*",
     ];
   }
+
+  /**
+   * Get a valid content replacement mode
+   *
+   * @return string
+   *   A valid mode for content replacement.
+   */
+  private function getValidContentReplacementMode() {
+    return 'trusted';
+  }
 }
```

Comment:
The way you unit and functional tested the new feature makes me want to bear-hug you. So good!